.github/workflows/areyouok-links.yml

```diff
@@ -0,0 +1,20 @@
+name: Audit Links using AreYouOk
+on:
+  workflow_dispatch:
+  schedule:
+    # At 06:00 AM, every 15 days
+    - cron:  '0 5 */15 * *'
+      
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download & Run AreYouOk
+        run: |
+          wget -q https://github.com/Bhupesh-V/areyouok/releases/latest/download/areyouok-linux-amd64
+          chmod +x areyouok-linux-amd64
+          ./areyouok-linux-amd64 -i=_layouts,.git,_site,build.py,USAGE.md -r=github
+      - name: Create issue if broken links found
+        uses: peter-evans/create-issue-from-file@v2
```

Comment:
I guess we accidentally removed this code?
```yaml
      uses: peter-evans/create-issue-from-file@v2
       with:
          title: AreYouOk URL Health Report
          content-filepath: ./report.github
```

Reply:
@HanakoK9 I guess this is the error you were talking about. Take a look at it & fix it with the suggestions provided above.